.github/workflows/release.yml

```diff
@@ -40,14 +40,11 @@ jobs:
           ./scripts/tag-submodules.sh ${tag}
 
       - name: 'Create Release Notes'
+        uses: sifive/action-release-notes@v1
```

Comment:
Here the release version will be enforce by the tag format defined above.
I assume any entity/person can uses this action, does not need to conform to our format for their release?

Reply:
Yeah, the action accepts any tag you want to provide it.